tests/unit/authors/test_authors_receivers.py

```diff
@@ -63,6 +73,93 @@ def test_name_variations():
             'Richard Ellis'])
 
 
+@httpretty.activate
+def test_phonetic_block_generation_ascii():
+    from flask import current_app
```

Comment:
Yeah, you should tell the test to use the `app` fixture in `tests/unit/conftest.py` by adding `app` as a parameter to every test.